src/main/java/org/jenkinsci/plugins/workflow/support/pickles/ExecutorPickle.java

```diff
@@ -53,6 +61,10 @@
 
     private final Queue.Task task;
 
+    private final boolean isEphemeral;
```

Comment:
Will be curious to see what others think of this (or if they think it's not worth tracking).

Reply:
@jglick It appears this behavior is very much unsafe due to the use of the Swarm Plugin -- please see https://issues.jenkins-ci.org/browse/JENKINS-36013?focusedCommentId=311834&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-311834  -- we might argue that perhaps this is not a correct use of EphemeralNode but in any case the fix in DurableTask is trivial.